### DISCUSSION TITLE
Is it possible to set a dividing line in completion item list?

### DISCUSSION THREAD

**Discussion by KrabsWong** [microsoft/vscode-discussions#408] [Extension Development QnA] [answered]
I am developing an extension for user to do Code Review, when user type some specific character, there will be an completion item list shown up(like the issue list in the comment dialog of Github pull request extension)

Since the data in the list have different meaning, I set different completionItem kind and also add detail for each one.
What I wanna more is add a dividing line between different completion item. (Like a group)

It, maybe, like the demo below

<img width="1357" alt="image" src="https://user-images.githubusercontent.com/12368943/211997655-3c9d4b36-5fc5-4054-8a6c-41f80590bcb1.png">

I didn't find any property or api can support this kind of feature, so any way to implement it? or maybe no way so far.

**Comment by isidorn** [comment, marked as answer]
This is not possible right now. I suggest to use different suggestion icons for a visual separation.

**Reply by KrabsWong** [reply]
> This is not possible right now. I suggest to use different suggestion icons for a visual separation.

Any plan to support it?
Since I also didn't found a way to custom an icon, there is an enumeration for all the build-in icon.

**Reply by isidorn** [reply]
No plans right now.
Custom icons - not sure how far you can go with Product Icon Themes and if they allow changing the Completion Items icons.
